src/CMakeLists.txt

```diff
@@ -358,7 +358,7 @@ function(make_symlink orig link)
 	# Create the symlink (platform-dependent):
 	message("Creating symlink, orig = ${orig}; link = ${link}")
 	if (CMAKE_HOST_UNIX)
-		set(command ln -s ${orig} ${link})
+		set(command ln -fs ${orig} ${link})
```

Comment:
This should not be needed, there's an existence check on line 354, doesn't that work?